controllers/data_base/uploader.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Uploader # :nodoc:
+  PATH = './data_base/database.yaml'.freeze
+
+  def save_to_db
```

Comment:
modules(mixins) shouldn't work with instance variables, try to move them to args or smth like that